modules/auxiliary/gather/ldap_esc_vulnerable_cert_finder.rb

```diff
@@ -201,7 +238,7 @@ def query_ldap_server_certificates(esc_raw_filter, esc_name, notes: [])
           ca_servers_n_enrollment_sids: {},
           manager_approval: ([entry[%s(mspki-enrollment-flag)].first.to_i].pack('l').unpack1('L') & Rex::Proto::MsCrtd::CT_FLAG_PEND_ALL_REQUESTS) != 0,
           required_signatures: [entry[%s(mspki-ra-signature)].first.to_i].pack('l').unpack1('L'),
-          notes: notes
+          notes: notes.dup
```

Comment:
This was a bug found during testing. 

Some `certificate_symbol`'s within the `@vuln_certificate_details` hash were being created with the same `notes` array. This resulted in undesired behaviour when attempting to update the `[:notes]` key value pair of one cert, multiple certs were getting updated because they were previously referencing the same array.